docs/extend/plugins_network.md

```diff
@@ -11,30 +11,28 @@ weight=-1
 
 # Docker network driver plugins
 
-Docker supports network driver plugins via
-[LibNetwork](https://github.com/docker/libnetwork). Network driver plugins are
-implemented as "remote drivers" for LibNetwork, which shares plugin
-infrastructure with Docker. In effect this means that network driver plugins
-are activated in the same way as other plugins, and use the same kind of
-protocol.
+Docker supports network driver plugins via LibNetwork project. Network driver
+plugins are implemented as "remote drivers" for LibNetwork, which shares plugin
+infrastructure with Docker. In effect this means that network driver plugins are
+activated in the same way as other plugins, and use the same kind of protocol.
 
 ## Using network driver plugins
 
-The means of installing and running a network driver plugin will depend on the
+The means of installing and running a network driver plugin depend on the
```

Comment:
The means depend on --- is the correct noun/verb agreement. The clause "of installing and running a network driver plugin" clause intervening is throwing you for a loop.